Mining tutorial (#9163)
endless-sky/endless-sky: data/human/intro missions.txt

			`	"I've got an idea for something you might want to try," says James. "Meet me in the spaceport bar in a little while."`

mission "Intro [3 Interceptor]"
	priority
	name "Mine at <planet>"
	description "Mine asteroids at <destination>."
	passengers 1
	source "Ingot"
	destination "New Sahara"
	to offer
		has "Intro [2 Interceptor]: done"
	
	on offer
		conversation
			`James waves you over to his table when you enter the bar. He's sitting with another man with the local star map spread out between them. The steady patter of micrometeorites against the roof reminds you of rain on the factory roof in New Boston.`
			`	"This is Captain Marshall," James says as you pull up a chair. "He mines in this part of space and says there's a good system nearby if you're willing to give it a try."`
			choice
				`	"I'm up for a challenge."`
					goto mining
				`	"I'm ready to head out on my own."`
					decline
	
			label "mining"
			`	"That's good to hear," Captain Marshall says with a smile. "Mining without an asteroid scanner is a test of your piloting skills, but one that pays well."`

> 			`	"The first challenge," he continues, "will be finding which asteroids are valuable. The worthless asteroids are either brown and smooth or gray and spiky with no apparent orbit. It doesn't matter how long you shoot them, they won't be destroyed. Valuable asteroids will always orbit the star and have a range of colors and surface textures. Titanium tends to be a bit larger, with a pitted surface. Lead is light gray and has a boxy shape. Silicon has a bit of a greenish cast and looks like it's been melted in places. I increase the resolution on my control displays to make identification and targeting easier."`

Reviewer: ```suggestion
			`	"The first challenge," he continues, "will be finding which asteroids are valuable. The worthless asteroids are either brown and smooth or gray and spiky with no apparent orbit. It doesn't matter how long you shoot them, they won't be destroyed. Valuable asteroids will always orbit the star and have a range of colors and surface textures. Titanium tends to be a bit larger, with a pitted surface. Lead is light gray and has a boxy shape. Silicon has a bit of a greenish cast and looks like it's been melted in places. I increase the resolution on my control displays to make identification and targeting easier.`
```
I think there should be no ending comma if the person keeps talking.

			`	"Also remember that the more valuable the mineral the longer it takes to mine. Silicon breaks up quickly and you'll fill your hold in a hurry. Titanium is more work, but you make several times the number of credits for the effort."`

> 			`	Captain Marshall wishes you luck and heads out into the spaceport. You and James spend a few minutes going over the plan as you finish your drinks. "Leave your hold empty and fill it up before you land at <destination>. You can sell what you mine when we land. Remember there's no guaranteed payment in this kind of work. How much you make is based on the value of the minerals in your hold."`

Reviewer 2: We'd want to somehow add that you need to mine silicon to continue here. Maybe Marshall says something like "You should probably start easy and bag yourself five tons of silicon."

Author: Done: "You should start off easy and mine five tons of silicon. If you've got the hang of it, you can fill your hold with more valuable minerals."

				accept
	

> 	on complete

Reviewer 2: This mission has no objective other than reach the destination, so it's not really a good tutorial given that you could finish the mission without actually mining. There should be a requirement to mine some number of a common asteroid in the destination system. Phecda has loads of silicon, so maybe require that you've mined 5 silicon to continue.
```suggestion
	on visit
		dialog `You've reached <destination>, but you don't have at least five tons of silicon in your cargo hold. Locate and mine silicon asteroids in this system before returning.`
	
	on complete
		require "Silicon" 5
```

		conversation

> 			`You land on <planet> and sell your cargo of minerals. You meet up with James and he asks you, "What did you think of mining?"`

Reviewer: You didn't sell your minerals; you may do that after the conversation.

			choice

> 				`	"I can't believe anyone would do that much work."`

Reviewer 2: This response is negative, but James response feels like you'd said something positive. Perhaps some affirmation from James that it can be difficult (which would also work regardless of your response). "Mining can be difficult work, but when you're starting off, it can be..."

Author: Done: James nods and says, "Chasing asteroids is always a challenge. But when you're starting off...

				`	"I love flying like that."`

> 			`	James nods and says, "When you're starting off, mining can be a fast way to build cash for your next ship. It's also a great way to explore the galaxy. Jobs always follow the trade routes and settled areas. Mining makes it profitable to explore the nooks and crannies of the hyperlanes. If the place you end up landing has an economy, they'll buy the minerals in your hold. Mainly though, it's about how you like to spend your time. Some captains like managing a trading fleet while others like to strike out on their own and fly their ship directly."`

Reviewer 2: ```suggestion
			`	James nods and says, "When you're starting off, mining can be a fast way to build cash for your next ship. It's also a great way to explore the galaxy. Jobs always follow the trade routes and settled areas. Mining makes it profitable to explore the nooks and crannies of the hyperlanes. If the place you end up landing has an economy, they'll buy the minerals in your hold. Mainly though, it's about how you like to spend your time. Some captains like managing a trading fleet while others like to strike out on their own and fly their ship directly.`
```

> 			`	"Mining can be complex compared to running jobs or trading commodities. Are there any questions you'd like to ask?"`

Reviewer 2: This "any questions" feels like it comes out of nowhere, as it suggests that James is knowledgeable in mining. I imagine he is, but in the previous conversation we just had someone else give us the ropes on mining instead of James telling us, possibly suggesting that James isn't too knowledgeable in the subject.

In the other intro missions, James often tells a story from his past and uses it as a way to give advice to the player. In "Intro [2 Freighter]" James tells a story about a time where he boarded a pirate ship and it didn't go well for him. Perhaps he says something here about how he tried his hand in combat while in an interceptor and nearly got himself killed, so he decided to stay as far from combat as he could and spent a few months mining in safer systems, meaning he knows a thing or two about mining. Then he asks if you have any questions that he'd be able to answer.

Author: Adjusted to: "Mining can be complex compared to running jobs or trading commodities. I learned that first hand after pirates disabled my ship when I was starting out. "

This is a bit tricky because James says that he switched out his beam lasers for missiles earlier so he probably didn't mine in his own ship.

			label choices
			choice
				`	"What equipment do I need to mine?"`
					goto equipment

> 				`	"Can my fleet help me mine?"`

Reviewer: ```suggestion
				`	"Can a fleet help me mine?"`
```
It feels better to me to say this because you probably don't have a fleet yet.

					goto fleet
				`	"Where are safe places to mine?"`
					goto safety
				`	"How do you deal with pirates?"`
					goto pirates
				`	"How can I check the value of minerals?"`
					goto value

> 				`	"I don't have any more questions."`

Reviewer 2: Doesn't make sense if this is your first choice, as you didn't ask any questions to begin with. Perhaps instead say something like "I think I have enough information."

Author: Implemented as suggested.

					goto end

			label equipment
			`	"Well, you need a weapon that does hull damage," James says. "A beam laser will get the job done, but it will take awhile. The most valuable tool is an asteroid scanner. They're expensive, but they connect into your navigation and targeting systems to point you towards minable asteroids. You can also set them to find the most valuable asteroids which increases your profitability per run. After that, a tactical scanner will give you range to target and show how much damage you're doing. An afterburner is good for chasing down fast moving asteroids or getting away from pirates without leaving the system. A ramscoop will let you refuel while mining. You can extend your range and be ready for a quick escape."`
			choice
				`	"I can't fit all that on my ship!"`
					goto shipmods
				`	"That all makes sense."`
					goto more
			
			label shipmods

> 			`	James gives you a knowing smile as he answers, "Not in a factory build, you won't. But you'd be surprised how much space you can free up even in a Sparrow if you're willing to flexible about health and safety. Do you really need that much power? If you've got an outfitter on the planet where you're mining, do you need a hyperdrive, or even a shield regenerator? Maybe a cargo expansion would be more use." He pauses briefly, perhaps considering what liability he's opened himself up for, then adds, "I'm not saying you should, of course, but you have options.`

Reviewer 2: ```suggestion
			`	James gives you a knowing smile as he answers, "Not in a factory build, you won't. But you'd be surprised how much space you can free up even in a Sparrow if you're willing to flexible about health and safety. Do you really need that much power? If you've got an outfitter on the planet where you're mining, do you need a hyperdrive, or even a shield generator? Maybe a cargo expansion would be more use." He pauses briefly, perhaps considering what liability he's opened himself up for, then adds, "I'm not saying you should, of course, but you have options.`
```

> 			`	James gives you a knowing smile as he answers, "Not in a factory build, you won't. But you'd be surprised how much space you can free up even in a Sparrow if you're willing to flexible about health and safety. Do you really need that much power? If you've got an outfitter on the planet where you're mining, do you need a hyperdrive, or even a shield regenerator? Maybe a cargo expansion would be more use." Seeing the look on your face he holds up his hands. "I'm not saying you should, of course, but you have options."`

Reviewer: > Seeing the look on your face he holds up his hands. "I'm not saying you should, of course, but you have options."

You probably shouldn't say you have a look on your face; that assumes stuff about the player. And for me, that's a wrong assumption. I totally do that!

Author: Removed  the player's reaction:  He pauses briefly, perhaps considering what liability he's opened himself up for, then adds, "I'm not saying you should, of course, but you have options.

> 			`	James gives you a knowing smile as he answers, "Not in a factory build, you won't. But you'd be surprised how much space you can free up even in a Sparrow if you're willing to flexible about health and safety. Do you really need that much power? If you've got an outfitter on the planet where you're mining, do you need a hyperdrive, or even a shield regenerator? Maybe a cargo expansion would be more use." Seeing the look on your face he holds up his hands. "I'm not saying you should, of course, but you have options."`

Reviewer: ```suggestion
			`	James gives you a knowing smile as he answers, "Not in a factory build, you won't. But you'd be surprised how much space you can free up even in a Sparrow if you're willing to flexible about health and safety. Do you really need that much power? If you've got an outfitter on the planet where you're mining, do you need a hyperdrive, or even a shield regenerator? Maybe a cargo expansion would be more use." Seeing the look on your face he holds up his hands. "I'm not saying you should, of course, but you have options.`
```

				goto more

			label fleet

> 			`	"If you have a fleet, you can target an asteroid and they will all fire on it and then pick up the minerals. You can work much more quickly and harvest more in a run. If you have a carrier, you can have fighters mine and return the minerals to your hold."`

Reviewer: ```suggestion
			`	"If you have a fleet, you can target an asteroid and they will all fire on it and then pick up the minerals. You can work much more quickly and harvest more in a run. If you have a carrier, you can have fighters mine and return the minerals to your hold.`
```

				goto more

			label safety

> 			`	James takes a moment to think before responding, "Go where pirates aren't welcome," he says. "The Navy heavily patrols the Paradise Worlds and pirates don't last long in the Deep. There are some systems in the dirt belt with a strong Navy presence and high quality asteroids. You could also go to the Quarg systems and you won't be bothered. Just know that danger and opportunity go together so you'll often find lucrative mining in uninhabited spaces and around pirate worlds.`

Reviewer 2: ```suggestion
			`	James takes a moment to think before responding, "Go where pirates aren't welcome," he says. "The Navy heavily patrols the Paradise Worlds and pirates don't last long in the Deep. There are some systems in the Dirt Belt with a strong Navy presence and high quality asteroids. You could also go to the Quarg systems and you won't be bothered. But know that danger and opportunity go together, so you'll often find lucrative mining in uninhabited spaces and around pirate worlds.`
```

> 			`	James takes a moment to think before responding, "Go where pirates aren't welcome," he says. "The Navy heavily patrols the Paradise Worlds and pirates don't last long in the Deep. There are some systems in the dirt belt with a strong navy presence and high quality asteroids. You could also go to the Quarg systems and you won't be bothered. Just know that danger and opportunity go together so you'll often find lucrative mining in uninhabited spaces and around pirate worlds."`

Reviewer: ```suggestion
			`	James takes a moment to think before responding, "Go where pirates aren't welcome," he says. "The Navy heavily patrols the Paradise Worlds and pirates don't last long in the Deep. There are some systems in the dirt belt with a strong Navy presence and high quality asteroids. You could also go to the Quarg systems and you won't be bothered. Just know that danger and opportunity go together so you'll often find lucrative mining in uninhabited spaces and around pirate worlds.`
```

				goto more

			label pirates

> 			`	"You can always bribe them when you get into the system and they'll leave you alone. If the minerals are good enough and you have enough cargo capacity you'll still turn a profit. If you have a fast ship, you can just run away. Most pirates are lazy and will turn back if you head to the edges of the system. Of course, with enough of a fleet you just fight them off." He shakes his head. "Most pirates would be richer and live longer if they just mined instead of extorting bribes and cargo."`

Reviewer: ```suggestion
			`	"You can always bribe them when you get into the system and they'll leave you alone. If the minerals are good enough and you have enough cargo capacity you'll still turn a profit. If you have a fast ship, you can just run away. Most pirates are lazy and will turn back if you head to the edges of the system. Of course, with enough of a fleet you just fight them off." He shakes his head. "Most pirates would be richer and live longer if they just mined instead of extorting bribes and cargo.`
```

				goto more
			
			label value

> 			`	"If you check the outfitter section of your star map you will find minerals listed at the very bottom. Selecting a mineral will show you systems where your asteroid scanner has detected them.`

Reviewer 2: Feels a bit too out of universe. I'd just say something like "that's something you have to learn with experience."

Author: Adjusted to: "You will learn this by experience, but if the planet has an outfitter, you can check the price there before you sell."

> 			`	"If you check the outfitter section of your star map you will find minerals listed at the very bottom. Selecting a mineral will show you systems where your asteroid scanner has detected them."`

Reviewer: ```suggestion
			`	"If you check the outfitter section of your star map you will find minerals listed at the very bottom. Selecting a mineral will show you systems where your asteroid scanner has detected them.`
```

				goto more

			label more
			`	"Anything else you want to ask me?"`
				goto choices
			
			label end
			`	"If you're willing to take me on one more journey," James says, "meet me in the spaceport bar in a little while."`

mission "Intro [4 Interceptor]"
	priority
	name "Bring James to <planet>"
	description "Drop James off on <destination>, where he has a spot reserved in a retirement home."
	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I found you another mission, but it looks like you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	source "New Sahara"
	destination "Hestia"
	to offer

> 		has "Intro [3 Interceptor]: done"

Author: Implemented as suggested.

				`	"Sure, but I can't wait until I've got a ship big enough to stand up to pirates instead of running from them."`
				`	"Well, maybe I should've bought a freighter instead."`
			`	"Better start saving up then," he says. "I know a number of old Sparrow pilots who made their first million credits mining asteroids before they could afford a more combat-worthy ship. They had to look around for a while before finding a system with any asteroids worth mining, but it was worth it. Anyway, let's go get your payment." You walk out of the ship and sign the papers to hand off the <commodity> to the dock workers on <planet>. They hand you a credit chip worth <payment>.`
			`	"I've got an idea for something you might want to try," says James. "Meet me in the spaceport bar in a little while."`

mission "Intro [2.1 Interceptor]"
	priority

> 	name "Mine at <planet>"

Reviewer 3: ```suggestion
	name "Go to the <system> system and mine 5 tons of Silicon, then land on <planet>."
```

Reviewer 2: This is the name field, not the description field.

Reviewer 3: ```suggestion
	name "Mine asteroids in <system>"
```

> 	description "Mine asteroids at <destination>."

Reviewer 3: ```suggestion
	description "Go to the <system> system and mine 5 tons of Silicon, then land on <planet>."
```

	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I found you another mission, but it looks like you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	source "Ingot"
	destination "New Sahara"
	to offer
		has "Intro [2 Interceptor]: done"
	
	on offer
		conversation
			`James waves you over to his table when you enter the bar. He's sitting with another man with a local star map spread out between them. There's a steady patter of micrometeorites against the roof that reminds you of rain on the factory roof in New Boston.`
			`	"This is Captain Marshall," James says as you pull up a chair. "He mines in this part of space and says there's a good system nearby if you're willing to give it a try."`
			choice
				`	"I'm up for a challenge."`
					goto mining
				`	"I'm ready to head out on my own."`
					decline
	
			label "mining"

> 			`	"That's good to hear," Captain Marshall says with a smile. "Mining without an asteroid scanner is a test of your piloting skills, but one that pays well.`
> 			`	"The first challenge," he continues, "will be finding which asteroids are valuable. The worthless asteroids are either brown and smooth or gray and spiky with no apparent orbit. It doesn't matter how long you shoot them, they won't be destroyed. Valuable asteroids will always orbit the star and have a range of colors and surface textures. Titanium tends to be a bit larger, with a pitted surface. Lead is light gray and has a boxy shape. Silicon has a bit of a greenish cast and looks like it's been melted in places. I increase the resolution on my control displays to make identification and targeting easier.`
> 			`	"Also remember that the more valuable the mineral the longer it takes to mine. Silicon breaks up quickly and you'll fill your hold in a hurry. Titanium is more work, but you make several times the number of credits for the effort."`
> 			`	Captain Marshall wishes you luck and heads out into the spaceport. You and James spend a few minutes going over the plan as you finish your drinks. "You should start off easy and mine five tons of silicon. If you've got the hang of it, you can fill your hold with more valuable minerals. There's plenty of both at <destination>. Remember, there's no guaranteed payment in this kind of work; how much you make is based on the value of the minerals in your hold."`

Reviewer 2: While this is a tutorial, the dialog feels a bit too tutorial-y to me compared to the other missions with James. How's this sound? Also seems a bit odd to me that when James introduces Marshall, he says that Marshall knows some good places to mine nearby, but Marshall isn't actually the one to tell you where to go mind in this dialog right now.
```suggestion
			`	"That's good to hear," Captain Marshall says with a smile. "But be warned that mining is as easy as one might think. Especially without an asteroid scanner, mining can be just as much a test of your piloting skills as combat.`
			`	"The first challenge," he continues, "will be finding which asteroids are valuable. The worthless asteroids are either brown and smooth or gray and spiky with no apparent orbit. It doesn't matter how long you shoot them, they won't be destroyed. Valuable asteroids will always orbit the star and have a range of colors and surface textures. If you're not sure which are which and you don't have an asteroid scanner to tell you, keep a look out for other miners to see what they're firing at.`
			`	"Generally speaking, the more valuable minerals are tougher to mine, either because they're harder to break apart or they're less common and in more dangerous regions of the galaxy. Silicon breaks up quickly; you'll fill your hold in a hurry and the stuff is everywhere. Platinum or gold are more work; not as common in areas where the Navy watches your back, but you make several times the number of credits for the effort.`
			`	"I could go on, but I think you get the picture."`
			`	You and James chat with Marshall a while longer, and he suggests trying your hand at mining in <system>. "It's a great place. You could throw a dart in that system and half the time you'd hit an asteroid worth at least something, and I've never been bothered by pirates while I've been there."`
			`	"That does sound like a good place for a beginner pilot," James says. "How about we start small, <first>? Let's go mine at least five tons of silicon before landing on <planet> and we can call it a day."`
```

				accept
	
	on visit
		dialog `You've reached <destination>, but you don't have at least five tons of silicon in your cargo hold. Locate and mine silicon asteroids in this system before returning.`
	
	on complete
		require "Silicon" 5
		conversation
			`Once you've landed, you meet up with James and he asks you, "What did you think of mining?"`
			choice
				`	"I can't believe anyone would do that much work."`
				`	"I love flying like that."`

> 			`	James nods and says, "Chasing asteroids is always a challenge. But when you're starting off, mining can be a fast way to build cash for your next ship. It's also a great way to explore the galaxy. Jobs always follow the trade routes and settled areas. Mining makes it profitable to explore the nooks and crannies of the hyperlanes. If the place you end up landing has an economy, they'll buy the minerals in your hold. Mainly though, it's about how you like to spend your time. Some captains like managing a trading fleet while others like to strike out on their own and fly their ship directly.`

Reviewer: > `But when you're starting off, mining can be a fast way to build cash for your next ship.`

A while back when I was using a bunch of plugins, I made a lot of money mining platinum/gold/uranium in the "northeast" pirate area. (One plugin was Mega Freight; I was using the Badger I think) (Though I did cheat to get my ship too)

> 			`	James nods and says, "Chasing asteroids is always a challenge. But when you're starting off, mining can be a fast way to build cash for your next ship. It's also a great way to explore the galaxy. Jobs always follow the trade routes and settled areas. Mining makes it profitable to explore the nooks and crannies of the hyperlanes. If the place you end up landing has an economy, they'll buy the minerals in your hold. Mainly though, it's about how you like to spend your time. Some captains like managing a trading fleet while others like to strike out on their own and fly their ship directly.`

Reviewer 2: ```suggestion
			`	James nods and says, "Chasing asteroids can be a challenge, but when you're starting off, mining can be a fast way to build cash for your next ship. It's also a great way to explore the galaxy. Jobs always follow the trade routes and settled areas. Mining makes it profitable to explore the nooks and crannies of the hyperlanes. If the place you end up landing has an economy, they'll buy the minerals in your hold. Mainly though, it's about how you like to spend your time. Some captains like managing a trading fleet while others like to strike out on their own.`
```

> 			`	"Mining can be complex compared to running jobs or trading commodities. I learned that first hand after pirates disabled my ship when I was starting out. I worked as crew on a mining ship while it was being repaired. Are there any questions you'd like to ask?"`

Author: This was added to provide context for James' knowledge of mining based on previous feedback.

> 			`	"Mining can be complex compared to running jobs or trading commodities. I learned that first hand after pirates disabled my ship when I was starting out. I worked as crew on a mining ship while it was being repaired. Are there any questions you'd like to ask?"`

Reviewer 2: ```suggestion
			`	"Mining can be complex compared to running jobs or trading commodities. I learned that first hand after pirates disabled my ship when I was starting out and I worked aboard a mining ship while it was being repaired. Now that was a while back so I might be a little rusty, but I could try at answering any other questions you might have about mining."`
```

			label choices
			choice
				`	"What equipment do I need to mine?"`
					goto equipment

> 				`	"Can a fleet help me mine?"`
> 					goto fleet

Reviewer 2: We already have the actual help dialogs that talk about fleet mining, so I'd just drop this option.
```suggestion
```

> 				`	"Where are safe places to mine?"`
> 					goto safety

Reviewer 2: This is effectively already covered by the "how do I avoid pirates" question in the final mission.
```suggestion
```

				`	"How do you deal with pirates?"`
					goto pirates
				`	"How can I check the value of minerals?"`
					goto value
				`	"I think I have enough information."`
					goto end

			label equipment

> 			`	"Well, you need a weapon that does hull damage," James says. "A beam laser will get the job done, but it will take awhile. The most valuable tool is an asteroid scanner. They're expensive, but they connect into your navigation and targeting systems to point you towards minable asteroids. You can also set them to find the most valuable asteroids which increases your profitability per run. After that, a tactical scanner will give you range to target and show how much damage you're doing. An afterburner is good for chasing down fast moving asteroids or getting away from pirates without leaving the system. A ramscoop will let you refuel while mining. You can extend your range and be ready for a quick escape.`

Reviewer 2: ```suggestion
			`	"Well, you need a weapon that does hull damage," James says. "A beam laser will get the job done, but it will take awhile. The most valuable tool is an asteroid scanner. They're expensive, but they connect into your navigation and targeting systems to point you towards minable asteroids. You can also set them to find the most valuable asteroids which increases your profitability per run. After that, a tactical scanner will give you range to target and show how much damage you're doing. An afterburner is good for chasing down fast moving asteroids or getting away from pirates without leaving the system. A ramscoop will let you refuel while mining, can extend your exploration range, and allow you to be ready for a quick escape."`
```

			choice
				`	"I can't fit all that on my ship!"`
					goto shipmods
				`	"That all makes sense."`
					goto more
			
			label shipmods

> 			`	James gives you a knowing smile as he answers, "Not in a factory build, you won't. But you'd be surprised how much space you can free up even in a Sparrow if you're willing to flexible about health and safety. Do you really need that much power? If you've got an outfitter on the planet where you're mining, do you need a hyperdrive, or even a shield generator? Maybe a cargo expansion would be more use." He pauses briefly, perhaps considering what liability he's opened himself up for, then adds, "I'm not saying you should, of course, but you have options.`

Reviewer: Just as a note, this isn't a ridiculous suggestion to me; that's what I do in a mining play-through.
1. Do tutorial.
2. Sell unused outfits at Hestia.
3. Mine copper and iron.

				goto more

> 			label fleet
> 			`	"If you have a fleet, you can target an asteroid and they will all fire on it and then pick up the minerals. You can work much more quickly and harvest more in a run. If you have a carrier, you can have fighters mine and return the minerals to your hold.`
> 				goto more

Reviewer 2: ```suggestion
```

> 
> 			label safety
> 			`	James takes a moment to think before responding, "Go where pirates aren't welcome," he says. "The Navy heavily patrols the Paradise Worlds and pirates don't last long in the Deep. There are some systems in the Dirt Belt with a strong Navy presence and high quality asteroids. You could also go to the Quarg systems and you won't be bothered. But know that danger and opportunity go together, so you'll often find lucrative mining in uninhabited spaces and around pirate worlds.`
> 				goto more
> 

Reviewer 2: ```suggestion
```

			label pirates

> 			`	"You can always bribe them when you get into the system and they'll leave you alone. If the minerals are good enough and you have enough cargo capacity you'll still turn a profit. If you have a fast ship, you can just run away. Most pirates are lazy and will turn back if you head to the edges of the system. Of course, with enough of a fleet you just fight them off." He shakes his head. "Most pirates would be richer and live longer if they just mined instead of extorting bribes and cargo.`

Reviewer 2: Let's not tell people to cheese the system fence in a tutorial.
```suggestion
			`	"You can always bribe them to leave you alone. If the minerals are good enough and you have enough cargo capacity, you'll still turn a profit. Or if you have a fast ship, you can just run away. Of course, with enough of a fleet you just fight them off." He shakes his head. "Most pirates would be richer and live longer if they just mined instead of extorting bribes and cargo.`
```

			`	"I've got an idea for something you might want to try," says James. "Meet me in the spaceport bar in a little while."`

mission "Intro [2.1 Interceptor]"
	priority
	name "Mine asteroids in <system>"
	description "Go to the <system> system and mine 5 tons of Silicon, then land on <planet>."
	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I think you'll find what I want to show you next very interesting, but you'll need to free up <capacity> first. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	source "Ingot"
	destination "New Sahara"
	to offer
		has "Intro [2 Interceptor]: done"
		not "Intro [3 Interceptor]: offered"
	
	on offer
		conversation

> 			`You find James waiting for you just outside the spaceport bar. Against the steady patter of micrometeorites against the roof he tells you, "Working here is one of the few jobs in the galaxy more dangerous than fighting pirates. And worse, because if a big meteorite decides to strike this station, there's basically nothing they can do about it except hope they see it coming." He turns to you with a grin. "But nonetheless, I think you might be interested in this idea."`

Reviewer 3: ```suggestion
			`You find James waiting for you just outside the spaceport bar. Against the steady patter of micrometeorites against the roof he tells you, "Working here on <origin> is one of the few jobs in the galaxy more dangerous than fighting pirates. And worse, because if a big meteorite decides to strike this station, there's basically nothing the miners can do about it except hope they see it coming." He turns to you with a grin. "But nonetheless, I think you might be interested in this idea."`
```
Reading this over, I think a new player could confuse the miners on Ingot with asteroid mining. So best to make it explicit that it's the miners on Ingot who are the ones in the firing line.

			`	James nods and says, "Chasing asteroids can be a challenge, but when you're starting off, mining can be a fast way to build cash for your next ship. It's also a great way to explore the galaxy. Jobs always follow the trade routes and settled areas. Mining makes it profitable to explore the nooks and crannies of the hyperlanes. If the place you end up landing has an economy, they'll buy the minerals in your hold. Mainly though, it's about how you like to spend your time. Some captains like managing a trading fleet while others like to strike out on their own.`
			`	"Mining can be complex compared to running jobs or trading commodities. I learned that first hand after pirates disabled my ship when I was starting out and I worked aboard a mining ship while it was being repaired. Now that was a while back so I might be a little rusty, but I could try at answering any other questions you might have about mining."`

			label choices
			choice
				`	"What equipment do I need to mine?"`
				`	"How do you deal with pirates?"`
					goto pirates
				`	"How can I check the value of minerals?"`
					goto value
				`	"I think I have enough information."`
					goto end

> 			`	"Well, you need a weapon that does hull damage," James says. "A Beam Laser will get the job done, but it will take a while. If you want to optimise your mining operations, however, you'll want a Mining Laser. Not only does it cut through asteroids faster, it's also more precise and preserves more minerals when the asteroid breaks apart. Some miners like to have a bit of combat flexibility by mining with Heavy Lasers, however.`

Reviewer 3: ```suggestion
			`	"Well, you need a weapon that does hull damage," James says. "A Beam Laser will get the job done, but it will take a while. If you want to optimize your mining operations, however, you'll want a Mining Laser. Not only does it cut through asteroids faster, it's also more precise and preserves more minerals when the asteroid breaks apart. Some miners like to have a bit of combat flexibility by mining with Heavy Lasers, however.`
```
You guys and your 'murican spelling.

				goto more

			label pirates
			`	"You can always bribe them to leave you alone. If the minerals are good enough and you have enough cargo capacity, you'll still turn a profit. Or if you have a fast ship, you can just run away. Of course, with enough of a fleet you just fight them off." He shakes his head. "Most pirates would be richer and live longer if they just mined instead of extorting merchants and plundering ships.`
				goto more
			
			label value
			`	"You will learn this by experience, but if the planet has an outfitter, you can check the price there before you sell.`

			label more
			`	"Anything else you want to ask me?"`
				goto choices
			
			label end

> 			`	"If you're willing to take me on one more journey," James says, "meet me in the spaceport bar in a little while."`

Reviewer 3: ```suggestion
			`	"If you're willing to take me on one more journey," James says, "meet me in the spaceport bar in a little while. Don't forget to sell off your minerals at the trading post."`
```
We don't tell the player where or how to sell the minerals, so we should tell them to go to the trade panel.

Reviewer 2: How about "trading post or outfitter"?

Reviewer 3: New Sahara doesn't have an outfitter, but I get your point. Another option may be adding a new choice dialogue up above asking James where to sell the minerals.

Reviewer 2: ```suggestion
`	"If you're willing to take me on one more journey," James says, "meet me in the spaceport bar in a little while. Don't forget to sell off your minerals at the trading post or, if there is one, the outfitter."`
```

Reviewer 3: ```suggestion
			`	"If you're willing to take me on one more journey," James says, "meet me in the spaceport bar in a little while. Don't forget to sell off your minerals at the trading post, or if there is one, the outfitter."`
```
If you *insist*. But I *insist* the comma has to go there.

				`	"Sure, but I can't wait until I've got a ship big enough to stand up to pirates instead of running from them."`
				`	"Well, maybe I should've bought a freighter instead."`
			`	"Better start saving up then," he says. "I know a number of old Sparrow pilots who made their first million credits mining asteroids before they could afford a more combat-worthy ship. They had to look around for a while before finding a system with any asteroids worth mining, but it was worth it. Anyway, let's go get your payment." You walk out of the ship and sign the papers to hand off the <commodity> to the dock workers on <planet>. They hand you a credit chip worth <payment>.`
			`	"I've got an idea for something you might want to try," says James. "Meet me in the spaceport bar in a little while."`

mission "Intro [2.1 Interceptor]"
	priority
	name "Mine asteroids in <system>"
	description "Go to the <system> system and mine 5 tons of Silicon, then land on <planet>."

> 	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I found you another mission, but it looks like you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`

Reviewer 3: Right.

	passengers 1
	source "Ingot"
	destination "New Sahara"
	to offer
		has "Intro [2 Interceptor]: done"
		not "Intro [3 Interceptor]: offered"
	
	on offer
		conversation

> 			`James waves you over to his table when you enter the bar. He's sitting with another man with a local star map spread out between them. There's a steady patter of micrometeorites against the roof that reminds you of rain on the factory roof in New Boston.`
> 			`	"This is Captain Marshall," James says as you pull up a chair. "I told him about your new Sparrow and how it'd be a good idea for you to get a taste of mining. He mines in this part of space and says there's a good system nearby if you're willing to give it a try."`
> 			choice
> 				`	"I'm willing to give it a shot."`
> 					goto mining
> 				`	"I'm ready to head out on my own."`
> 					decline

Reviewer 3: ```suggestion
			`You find James waiting for you just outside the spaceport bar. Against the steady patter of micrometeorites against the roof he tells you, "Working here is one of the few jobs in the galaxy more dangerous than fighting pirates. And worse, because if a big meteorite decides to strike this station, there's basically nothing they can do about it except hope they see it coming." He turns to you with a grin. "But nonetheless, I think you might be interested in this idea."`
			`	James leads you down the side of the bar to a booth where a man with a half-eaten hamburger and a plate of fries is sitting, busily examining a star map of the local sector. He notices your arrival and stands up to greet you. "<first>, this is Captain Marshall," James says as you shake his hand. "I told him about your new ship and how it'd be a good idea for you to get a taste of asteroid mining. He mines in this part of space and says there's a good system nearby if you're willing to give it a try."`
			choice
				`	"I'll give it a shot."`
					goto mining
				`	"I think I'm ready to head out on my own."`
					decline
```
I miss this little tidbit, and also it lines up with the blocked message if James meets you and then walks into the bar to greet Marshall.

	
			label "mining"

> 			`	"That's good to hear," Captain Marshall says with a smile. "But be warned that mining is as easy as one might think. Especially without an asteroid scanner, mining can be just as much a test of your piloting skills as combat.`

Reviewer 4: ```suggestion
			`	"That's good to hear," Captain Marshall says with a smile. "But be warned that mining isn't as easy as one might think. Especially without an asteroid scanner, mining can be just as much a test of your piloting skills as combat.`
```
I think thats what was intended?

			`	"The first challenge," he continues, "will be finding which asteroids are valuable. The worthless asteroids are either brown and smooth or gray and spiky with no apparent orbit. It doesn't matter how long you shoot them, they won't be destroyed. Valuable asteroids will always orbit the star and have a range of colors and surface textures. If you're not sure which are which and you don't have an asteroid scanner to tell you, keep a look out for other miners to see what they're firing at.`

> 			`	"Generally speaking, the more valuable minerals are tougher to mine, either because they're harder to break apart or they're less common and in more dangerous regions of the galaxy. Silicon breaks up quickly; you'll fill your hold in a hurry and the stuff is everywhere. Platinum or gold are more work; not as common in areas where the Navy watches your back, but you make several times the number of credits for the effort.`

Reviewer 3: ```suggestion
			`	"Generally speaking, the more valuable minerals are tougher to mine, either because they're harder to break apart, or they're less common and in more dangerous regions of the galaxy. Silicon breaks up quickly; you'll fill your hold in a hurry and the stuff is everywhere. Platinum or gold asteroids are more work; they're not as common in areas where the Navy watches your back, but you make several times the number of credits for the effort.`
```

			`Once you've landed, you meet up with James and he asks you, "What did you think of mining?"`
			choice
				`	"I can't believe anyone would do that much work."`
				`	"I love flying like that."`
			`	James nods and says, "Chasing asteroids can be a challenge, but when you're starting off, mining can be a fast way to build cash for your next ship. It's also a great way to explore the galaxy. Jobs always follow the trade routes and settled areas. Mining makes it profitable to explore the nooks and crannies of the hyperlanes. If the place you end up landing has an economy, they'll buy the minerals in your hold. Mainly though, it's about how you like to spend your time. Some captains like managing a trading fleet while others like to strike out on their own.`
			`	"Mining can be complex compared to running jobs or trading commodities. I learned that first hand after pirates disabled my ship when I was starting out and I worked aboard a mining ship while it was being repaired. Now that was a while back so I might be a little rusty, but I could try at answering any other questions you might have about mining."`

			label choices
			choice
				`	"What equipment do I need to mine?"`

> 					goto equipment
> 				`	"How do you deal with pirates?"`
> 					goto pirates
> 				`	"How can I check the value of minerals?"`
> 					goto value
> 				`	"I think I have enough information."`
> 					goto end
> 
> 			label equipment

Reviewer 3: ```suggestion
				`	"How do you deal with pirates?"`
					goto pirates
				`	"How can I check the value of minerals?"`
					goto value
				`	"I think I have enough information."`
					goto end

```

			`	"Well, you need a weapon that does hull damage," James says. "A beam laser will get the job done, but it will take awhile. The most valuable tool is an asteroid scanner. They're expensive, but they connect into your navigation and targeting systems to point you towards minable asteroids. You can also set them to find the most valuable asteroids which increases your profitability per run. After that, a tactical scanner will give you range to target and show how much damage you're doing. An afterburner is good for chasing down fast moving asteroids or getting away from pirates without leaving the system. A ramscoop will let you refuel while mining, can extend your exploration range, and allow you to be ready for a quick escape."`
			choice
				`	"I can't fit all that on my ship!"`
					goto shipmods
				`	"That all makes sense."`
					goto more
			
			label shipmods

> 			`	James gives you a knowing smile as he answers, "Not in a factory build, you won't. But you'd be surprised how much space you can free up even in a Sparrow if you're willing to flexible about health and safety. Do you really need that much power? If you've got an outfitter on the planet where you're mining, do you need a hyperdrive, or even a shield generator? Maybe a cargo expansion would be more use." He pauses briefly, perhaps considering what liability he's opened himself up for, then adds, "I'm not saying you should, of course, but you have options.`

Reviewer 3: ```suggestion
			`	James gives you a knowing smile as he answers, "Not in a factory build, you won't. But you'd be surprised how much space you can free up even in a Sparrow if you're willing to flexible. Do you really need that much power? If you've got an outfitter on the planet where you're mining, do you need a hyperdrive, or even a shield generator? Maybe a cargo expansion would be of more use." He pauses briefly, perhaps considering what liability he's opened himself up for, then adds, "I'm not saying you should, of course, but you have options.`
```
You still have the shields to protect you, and removing your hyperdrive is not really anything to do with health and safety. But maybe you could leave the `health and safety` part in anyway.
```suggestion
			`	James gives you a knowing smile as he answers, "Not in a factory build, you won't. But you'd be surprised how much space you can free up even in a Sparrow if you're willing to flexible about health and safety. Do you really need that much power? If you've got an outfitter on the planet where you're mining, do you need a hyperdrive, or even a shield generator? Maybe a cargo expansion would be of more use." He pauses briefly, perhaps considering what liability he's opened himself up for, then adds, "I'm not saying you should, of course, but you have options.`
```

Reviewer 2: Just noticed is should be "be flexible".

Reviewer 3: Man, my brain just filled that one in like it had been there all along.

				goto more

			label pirates

> 			`	"You can always bribe them to leave you alone. If the minerals are good enough and you have enough cargo capacity, you'll still turn a profit. Or if you have a fast ship, you can just run away. Of course, with enough of a fleet you just fight them off." He shakes his head. "Most pirates would be richer and live longer if they just mined instead of extorting bribes and cargo.`

Reviewer 3: ```suggestion
			`	"You can always bribe them to leave you alone. If the minerals are good enough and you have enough cargo capacity, you'll still turn a profit. Or if you have a fast ship, you can just run away. Of course, with enough of a fleet you just fight them off." He shakes his head. "Most pirates would be richer and live longer if they just mined instead of extorting merchants and plundering ships.`
```
Seems to make a little bit more sense, again probably optional.

				goto more
			
			label value
			`	"You will learn this by experience, but if the planet has an outfitter, you can check the price there before you sell.`

> 				goto more

Reviewer 3: ```suggestion
```

	description "Drop James off on <destination>, where he has a spot reserved in a retirement home."
	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I found you another mission, but it looks like you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	source "New Sahara"
	destination "Hestia"
	to offer
		has "Intro [2.1 Interceptor]: done"
	
	on offer
		conversation

> 			`You find James sitting at the spaceport's tiny bar, sipping a beer while watching the large displays that show the landing pads and the irrigated fields beyond. The displays are supposed to make the underground building feel less claustrophobic. "This is one of the least habitable planets in the galaxy. But if it doesn't kill us instantly, then someone's tried to settle it." He pauses for a moment while gazing at one of the displays. You spot a sandstorm picking up in the far distance as the workers in the fields hastily finish their work before it arrives.`

Reviewer 3: ```suggestion
			`You find James sitting at the spaceport's tiny bar, sipping a beer while watching the large displays that show the landing pads and the irrigated fields beyond. The displays are supposed to make the underground building feel less claustrophobic.`
			`	"This is one of the least habitable planets in the galaxy. But if it doesn't kill us instantly, then someone's tried to settle it." He pauses for a moment while gazing at one of the displays. You spot a sandstorm picking up in the far distance as the workers in the fields hastily finish their work before it arrives.`
```
I think a line break here is warranted.

	description "Drop James off on <destination>, where he has a spot reserved in a retirement home."
	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I found you another mission, but it looks like you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	source "New Sahara"
	destination "Hestia"
	to offer
		has "Intro [2.1 Interceptor]: done"
	
	on offer
		conversation

> 			`You find James sitting at the spaceport's tiny bar, sipping a beer while watching the large displays that show the landing pads and the irrigated fields beyond. The displays are supposed to make the underground building feel less claustrophobic. "Care for a drink?" he asks.`

Reviewer 2: ```suggestion
			`You find James sitting at the spaceport's tiny bar, sipping a beer while watching the large displays that show the landing pads and the irrigated fields beyond. The displays are supposed to make the underground building feel less claustrophobic. "This is one of the least habitable planets in the galaxy. But if it doesn't kill us instantly, then someone's tried to settle it." He pauses for a moment while gazing at one of the displays. You spot a sandstorm picking up in the distance as the workers in the fields run for the safety of the underground spaceport.`
			`	"Care for a drink?" he asks.`
```

Reviewer 3: Looks good.

			choice
				`	"Sure."`

> 				`	"No thanks."`

Reviewer 3: Just below, the dialogue is significantly dampened now that it doesn't reference Ingot. There's all of 2000 people on New Sahara and nobody has mentioned anything about them.

Reviewer 2: Thoughts on the suggestion I added? (Might need to change stuff further below to line up with the impending sandstorm.)

		
	label end
	`	He hands you a credit chip worth <payment>, and says, "Best of luck to you, Captain. Don't hesitate to look me up if you're ever in the area. And remember to check the spaceports wherever you land. You never know who you'll find wandering around there."`

mission "Intro [Mining]"
	landing
	name "Explore Mining"

> 	description "Learn about mining."
> 	source "Sunracer"

Reviewer: ```suggestion
	invisible
```
You're just going to decline it at some point, it might as well be invisible. (Invisible might not go here though, I don't really know the order mission stuff usually goes.)

	to offer
		or
			has "ships: Interceptor"
			has "ships: Drone"
			has "ships: Fighter"
			has "ships: Light Warship"
			has "ships: Medium Warship"
			has "ships: Heavy Warship"
			has "ships: Superheavy"
	
	on offer
		conversation

> 			`	On your approach to <planet> you notice a large warehouse with private landing pads for the Clementine Metals Company. Someone there would know about mining asteroids. Would you like to ask them?`

Reviewer: ```suggestion
			`	As you approach <planet>, you spot a large warehouse belonging to the Clementine Metals Company. Maybe someone there could give you some tips on mining. Do you want to look around?`
```
I'm not sure how to say the private landing pad thing.

Reviewer 5: Restricted Berths should be a good substitute.

Reviewer: My problem isn't with the phrasing; it's with how you can tell their restricted without being awkward.

Reviewer 5: ATC telling you it's restricted airspace would be good.

			choice

> 				`	It would be nice to get more information.`

Reviewer: ```suggestion
				`	(Go in.)`
```

					goto mining

> 				`	I'm busy now, but maybe later.`

Reviewer: ```suggestion
				`	(Maybe later.)`
```

					defer

> 				`	I'm not interested.`

Reviewer: ```suggestion
				`	(No.)`
```

					decline
	
			label "mining"

> 			`	The warehouse is surprisingly well organized for a place that deals with raw ores. The receptionist tells you that the owner is due back shortly and points you to some picnic tables outside. You have a seat and watch the trucks loading and carting off various minerals. Soon enough a battered Aerie so covered in scorch marks and hull repairs that you can barely make out it's registration markings settles in next to you. When the hatch opens, a woman with close cut hair and a worn but crisp navy flight suit walks up to you. "I'm Captain Zinaida she says without preamble. You were wanting to asking about mining?" You nod your head and she says, "Well, go ahead."`

Reviewer: Why wouldn't it be well organized?

Reviewer 5: Missing Quotes.

Reviewer: That's what most of my comments are about, but what I meant was about the warehouse organization: `The warehouse is surprisingly well organized for a place that deals with raw ores.`.

Reviewer 5: ```suggestion
			`	The warehouse is surprisingly well organized for a place that deals with raw ores. The receptionist tells you that the owner is due back shortly and points you to some picnic tables outside. You have a seat and watch the trucks loading and carting off various minerals. Soon enough a battered Aerie so covered in scorch marks and hull repairs that you can barely make out its registration markings settles in next to you. When the hatch opens, a woman with close-cut hair and a worn but crisp navy flight suit walks up to you. "I'm Captain Zinaida," she says without preamble. "You were wanting to ask about mining?" You nod your head and she says, "Well, go ahead."`
```

Reviewer 5: > That's what most of my comments are about, but what I meant was about the warehouse organization: `The warehouse is surprisingly well organized for a place that deals with raw ores.`.

This, idk. Why wouldn't it be organised?

Reviewer: > > That's what most of my comments are about, but what I meant was about the warehouse organization: `The warehouse is surprisingly well organized for a place that deals with raw ores.`.
> 
> This, idk. Why wouldn't it be organised?

Sorry, I may be communicating badly. That's the question I'm trying to ask.

What about raw ores makes it unlikely for it to be organized?

> 			`	The warehouse is surprisingly well organized for a place that deals with raw ores. The receptionist tells you that the owner is due back shortly and points you to some picnic tables outside. You have a seat and watch the trucks loading and carting off various minerals. Soon enough a battered Aerie so covered in scorch marks and hull repairs that you can barely make out it's registration markings settles in next to you. When the hatch opens, a woman with close cut hair and a worn but crisp navy flight suit walks up to you. "I'm Captain Zinaida she says without preamble. You were wanting to asking about mining?" You nod your head and she says, "Well, go ahead."`

Reviewer: ```suggestion
			`	The warehouse is surprisingly well organized for a place that deals with raw ores. After you come up to the desk, the receptionist says that the owner is due shortly, and waves you off to some picnic tables. You have a seat and watch the trucks loading and carting off various minerals. Soon enough a battered Aerie, so covered in scorch marks and hull repairs that you can barely make out its registration markings, settles in next to you. When the hatch opens, a woman with close cut hair and a worn but crisp navy flight suit walks up to you. "I'm Captain Zinaida," she says without preamble. "You were wanting to asking about mining?" You nod your head and she says, "Well, ask away."`
```

			label choices
			choice
				`	"Why would I mine rather than run a trading fleet?"`
					goto whymine
				`	"What equipment do I need to mine?"`
					goto equipment
				`	"Where are safe places to mine?"`
					goto safety
				`	"How do you deal with pirates?"`
					goto pirates
				`	"Can I mine without and asteroid scanner?"`
					goto noscanner
				`	"How can I check the value of minerals?"`
					goto value

> 				`	"Were you in the navy?"`

Reviewer: ```suggestion
				`	"Were you in the Navy?"`
```

					goto navy

> 				`	"I can't think of any more questions."`

Reviewer: ```suggestion
				`	"I can't think of anything else."`
```
Just sounds better to me.

					goto end
			
			label whymine

> 			`	"When you're starting off, mining can be a fast way to build cash for your next ship. It's also a great way to explore the galaxy. Jobs always follow the trade routes and settled areas. Mining makes it profitable to explore the nooks and crannies of the hyperlanes. If the place you end up landing has an economy, they'll buy the minerals in your hold. Mainly though, it's about how you like to spend your time. Sure, you can get rich running a trading fleet and managing other captains, but miners get to fly their ship directly." The look she gives you is both assessing and challenging.`

Reviewer: ```suggestion
			`	"When you're starting off, mining can be a fast way to build cash for your next ship. It's also a great way to explore the galaxy. Jobs always follow the trade routes and settled areas. Mining makes it profitable to explore the nooks and crannies of the galaxy. If the place you end up landing has an economy, they'll buy the minerals in your hold. Mainly though, it's about how you like to spend your time. Sure, you can get rich running a trading fleet and managing other captains, but miners get to fly their ship directly." The look she gives you is both assessing and challenging.`
```

				goto more
			
			label equipment

> 			`	Captain Zinaida looks over the <ship> critically. "Well, you need a weapon that does hull damage. A beam laser will get the job done but it will take awhile. The most valuable tool is an asteroid scanner. They're expensive but they connect into your navigation and targeting systems to point you towards minable asteroids. You can also set them to find the most valuable asteroids which increases your profitability per run. After that, a tactical scanner will give you range to target and show how much damage you're doing. An afterburner is good for chasing down fast moving asteroids or getting away from pirates without leaving the system. If you have a fleet, you can target an asteroid and they will all fire on it and then pick up the minerals. You can work much more quickly and harvest more in a run. If you have a carrier, like Bertha there, you can have fighters mine and return the minerals to your hold.`

Reviewer: ```suggestion
			`	Captain Zinaida looks over the <ship> critically. "Well, you need a weapon that does hull damage. A beam laser will get the job done but it will take awhile. The most valuable tool is an asteroid scanner. They're expensive but they connect into your targeting systems to show you where mineable asteroids are. You can also set them to find the most valuable asteroids which increases your profitability per run. After that, a tactical scanner will give you range to target and show how much damage you're doing. An afterburner is good for chasing down fast moving asteroids or getting away from pirates without leaving the system. If you have a fleet, you can target an asteroid and they will all fire on it and then pick up the minerals. You can work much more quickly and harvest more in a run. If you have a carrier, like Bertha there, you can have fighters mine and return the minerals to your hold.`
```
This made it sound like you always had the pointing towards target setting on.

			choice
				`	"I can't fit all that on my ship!"`
					goto shipmods
				`	"That all makes sense."`
					goto more
			
			label shipmods

> 			`	She gives you a crafty smile before responding. "Not in a factory build, you won't. But you'd be surprised how much space you can free up even in a sparrow if you're willing to flexible about health and safety. Do you really need that much power? If you've got an outfitter on the planet, do you need a hyperdrive, or even a shield regenerator? Maybe a cargo expansion would be more use?" Seeing the look on your face she holds her hands up. "I'm not saying you should, of course, but you have options."`

Reviewer: Why am I thinking that's bad? When I start out mining, I sell my shield generator and hyperdrive on the planet James goes to and put as many cargo expansions as I can fit in on my ship.

Reviewer: And you don't need that quote:
```suggestion
			`	She gives you a crafty smile before responding. "Not in a factory build, you won't. But you'd be surprised how much space you can free up even in a sparrow if you're willing to flexible about health and safety. Do you really need that much power? If you've got an outfitter on the planet, do you need a hyperdrive, or even a shield regenerator? Maybe a cargo expansion would be more use?" Seeing the look on your face she holds her hands up. "I'm not saying you should, of course, but you have options.`
```

				goto more

			label safety

> 			`	The captain shrugs and you get the feeling she's disappointed in your question. "Go where pirates aren't welcome," she says. There are some systems in the dirt belt with a strong navy presence and high quality asteroids. Or you could go to the Quarg systems and you won't be bothered. Just know that danger and opportunity go together so you'll often find lucrative mining in uninhabited spaces and around pirate worlds.`

Reviewer: ```suggestion
			`	The captain shrugs and you get the feeling she's disappointed in your question. "Go where pirates aren't welcome," she says. There are some systems in the Dirt Belt with a strong Navy presence and high quality asteroids. Or you could go to the Quarg systems and you won't be bothered. Just know that danger and opportunity go together so you'll often find lucrative mining in uninhabited spaces and around pirate worlds.`
```

				goto more

			label pirates

> 			`	"You can always bribe them when you get into the system and they'll leave you alone. If the minerals are good enough and you have enough cargo capacity you'll still turn a profit. If you have a fast ship, you can just run away. Most pirates are lazy and will turn back if you head to the edges of the system. Of course, with enough of a fleet you just fight them off." She shakes her head. "Most pirates would be richer and live longer if they just mined instead of extorting bribes and cargo."`

Reviewer: ```suggestion
			`	"You can always bribe them when you get into the system and they'll leave you alone. If the minerals are good enough and you have enough cargo capacity you'll still turn a profit. If you have a fast ship, you can just run away. Most pirates are lazy and will turn back if you head to the edges of the system. Of course, with enough of a fleet you just fight them off." She shakes her head. "Most pirates would be richer and live longer if they just mined instead of extorting bribes and cargo.`
```

				goto more
			
			label noscanner

> 			`	The captain smiles as if you've offered her a tasty treat. "A good pilot can do it. The worthless asteroids are either brown and smooth or gray and spiky with no apparent orbit. Valuable asteroids will always orbit the star and have a range of colors and surface textures. You have to be skilled to chase it down while firing and a better one to pick the minerals up sometimes. The good news is that if you're in a good system, you'll be able to afford a scanner quickly.`

Reviewer: A better what?

Reviewer 6: ```suggestion
			`	The captain smiles as if you've offered her a tasty treat. "A good pilot can do it. The worthless asteroids are either brown and smooth or gray and spiky with no apparent orbit. Valuable asteroids will always orbit the star and have a range of colors and surface textures. You have to be a skilled pilot to chase it down while firing, and a better one to pick up any minerals you blast loose. The good news is that if you're in a good system, you'll be able to afford a scanner quickly.`
```

				goto more
			
			label value

> 			`	If you check the outfitter section of your star map you will find minerals listed at the very bottom. Selecting a mineral will show you systems where your asteroid scanner has detected them.`

Reviewer: ```suggestion
			`	"If you check the outfitter section of your star map you will find minerals listed at the very bottom. Selecting a mineral will show you systems where your asteroid scanner has detected them."`
```

				goto more

			label navy

> 			`	She nods. "I served thirty years. My last command was a cruiser out of Alnitak fighting pirates. The brass kept trying to kick me off the bridge and into a desk job, so I took my retirement and bought Bertha there so I could keep flying."`

Reviewer: ```suggestion
			`	She nods. "I served thirty years. My last command was a cruiser out of Alnitak fighting pirates. The brass kept trying to kick me off the bridge and into a desk job, so I took my retirement and bought Bertha there so I could keep flying.`
```

				goto more

			label more

> 			`	"Next question."`

Reviewer: ```suggestion
			`	"Next question?"`
```

				goto choices
			
			label end

> 			`	Captain Zinaida shakes your hand firmly and tells you, "If you don't take risks, you won't drink champagne. Good flying to you, Captain <last>.`

Reviewer: ```suggestion
			`	Captain Zinaida shakes your hand firmly and tells you, "If you don't take risks, you won't drink champagne. Good flying to you, Captain <last>."`
```

	description "Drop James off on <destination>, where he has a spot reserved in a retirement home."
	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I found you another mission, but it looks like you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	source "New Sahara"
	destination "Hestia"
	to offer
		has "Intro [2.1 Interceptor]: done"
	
	on offer
		conversation

> 			`You find James sitting at the spaceport's tiny bar, sipping a beer while watching the large displays that show the landing pads and the irrigated fields beyond. The displays are suppposed to make the underground building feel less claustrophobic, but living on a ship, that doesn't bother you much. "Care for a drink?" he asks.`

Reviewer 3: ```suggestion
			`You find James sitting at the spaceport's tiny bar, sipping a beer while watching the large displays that show the landing pads and the irrigated fields beyond. The displays are supposed to make the underground building feel less claustrophobic, but living on a ship, that doesn't bother you much. "Care for a drink?" he asks.`
```

Reviewer 3: Also, you've only lived on the ship for something like a week? Maybe remove that line?
